webapp/src/components/App/Settings/Tokens/dialog/TokenFormDialog/TokenForm.tsx

```diff
@@ -172,7 +157,9 @@ function TokenForm(props: Props) {
                   variant="contained"
                   disabled={!allowToAddPermission}
                   onClick={() => {
-                    append({ group: selectedGroup, type: RoleType.READER });
+                    if (allowToAddPermission) {
```

Comment:
I think it was an old code but it would be nice if you can rename `allowToAddPermission` -> `isAllowedToAddPermission` it's grammatically more correct, or `canAddPermission` more concise

Reply:
it's also on the other forms like `UserFormDialog`